yarn packages attract pentesters and malware

### Are you using XOA or XO from the sources?

XO from the sources

### Which release channel?

None

### Provide your commit number

_No response_

### Describe the bug

I am following instructions at https://docs.xen-orchestra.com/installation and receive different results every time I run yarn and yarn build, the latest error was "src/pages/host/[id].vue:20:26 - error TS2306: File '/root/xen-orchestra/node_modules/vue-router/vue-router-auto.d.ts' is not a module."

Of course when I "viewed" my "router" I'm being flooded with penetration testers.

<img width="1898" height="741" alt="Image" src="https://github.com/user-attachments/assets/dcea20f3-951a-49c5-9fce-794e76951576" />

### Error message

```Text

```

### To reproduce

1. Go to '...'
2. Click on '...'
3. Scroll down to '...'
4. See error


### Expected behavior

_No response_

### Screenshots

_No response_

### Node

20

### Hypervisor

n/a

### Additional context

_No response_

--- a/@xen-orchestra/web/src/pages/host/[id].vue
+++ b/@xen-orchestra/web/src/pages/host/[id].vue
@@ -17,7 +17,7 @@ import { useDefaultTab } from '@core/composables/default-tab.composable.ts'
 import { useUiStore } from '@core/stores/ui.store'
 import type { XoHost } from '@vates/types'
 import { useI18n } from 'vue-i18n'
-import { useRoute } from 'vue-router/auto'
+import { useRoute } from 'vue-router'
 
 useDefaultTab('/host/[id]', 'dashboard')
 

--- a/CHANGELOG.unreleased.md
+++ b/CHANGELOG.unreleased.md
@@ -15,6 +15,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- Fix build issue [#9455](https://github.com/vatesfr/xen-orchestra/issues/9455)
+
 ### Packages to release
 
 > When modifying a package, add it here with its release type.
@@ -30,5 +32,7 @@
 > Keep this list alphabetically ordered to avoid merge conflicts
 
 <!--packages-start-->
+
+- @xen-orchestra/web patch
 
 <!--packages-end-->

--- a/CHANGELOG.unreleased.md
+++ b/CHANGELOG.unreleased.md
@@ -15,7 +15,7 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
-- Fix build issue [#9455](https://github.com/vatesfr/xen-orchestra/issues/9455)
+- Fix build issue [#9455](https://github.com/vatesfr/xen-orchestra/issues/9455) (PR [#9482](https://github.com/vatesfr/xen-orchestra/pull/9482))
 
 ### Packages to release
 

